tests/testenvironment.proj

```diff
@@ -48,6 +50,8 @@
   <ItemDefinitionGroup>
     <TestEnvironment>
       <TieredCompilation>0</TieredCompilation>
+      <DbgEnableMiniDump>1</DbgEnableMiniDump> <!-- Enable minidumps for all scenarios -->
```

Comment:
Why set one of them selectively but not the other? They don't really do anything in Windows.